Library/Formula/crf++.rb

```diff
@@ -1,9 +1,8 @@
-require 'formula'
-
 class Crfxx < Formula
-  homepage 'http://code.google.com/p/crfpp/'
-  url 'https://crfpp.googlecode.com/files/CRF++-0.58.tar.gz'
-  sha1 '979a686a6d73d14cdd0c96a310888fb6bffd2e91'
+  homepage "https://taku910.github.io/crfpp/"
+  url "https://drive.google.com/uc?export=download&id=0B4y35FiV1wh7QVR6VXJ5dWExSTQ"
+  version "0.58"
+  sha256 "9d1c0a994f25a5025cede5e1d3a687ec98cd4949bfb2aae13f2a873a13259cb2"
 
   def install
     system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!